docs/UserGuide.adoc

```diff
@@ -139,11 +151,14 @@ Format: `deletemenu INDEX`
 * Deletes the menu item in that particular index
 
 Example: +
-viewallmenu +
-deletemenu 1 +
-System deletes food item in index 1 from the menu. +
-findmenu Cheese Fries +
-deletemenu 3 +
+`listmenutype main` +
+`deletemenu 1` +
+System deletes menu item in index 1 from the menu. +
+`listmenutype main` +
+`deletemenu 1` +
+System deletes the menu item in index 1 from the menu. +
+`findmenu Cheese Fries` +
+`deletemenu 3` +
 System deletes Cheese Fries in index 3 from the menu.
```

Comment:
Should be adjusted to:
... from the displayed menu of items found with `Cheese Fries` keyword